src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.ts

```diff
@@ -136,7 +136,14 @@ export class PoolFormComponent implements OnInit {
           validators: [Validators.required, Validators.min(1)]
         }),
         ecOverwrites: new FormControl(false),
-        compression: compressionForm
+        compression: compressionForm,
+        max_bytes: new FormControl(0, {
+          updateOn: 'blur'
+        }),
+        max_objects: new FormControl(0, {
+          validators: [Validators.min(0)],
+          updateOn: 'blur'
```

Comment:
I thinks this is already the default and can be removed.

Reply:
can we keep min validator because max_object is a number field on page?

Reply:
I was referring only to the `updateOn` line, the validator should stay.